codebuild/bin/test_libcrypto_interning.sh

```diff
@@ -15,27 +15,10 @@
 
 set -e
 
-
+source codebuild/bin/s2n_setup_env.sh
 source codebuild/bin/jobs.sh
 
-# build 2 different version of libcrypto to make it easy to break the application if
```

Comment:
Nice, thanks for updating this...

Reply:
So is this now just relying on our normal install of lots of different libcryptos? Should we do some sort of assertion that at least two libcryptos exist?

Reply:
The nice thing about how it was done is that it made it easy to just run the script and it made sure everything was built for it. Ideally we would preserve that functionality and make it easy to override it.

Reply:
The main problem was that it wasn't checking the test-deps install path, so it was re-installing the libcryptos that were already installed every run. And I was thinking that tests should always just use the dependencies installed on the docker image rather than downloading their own stuff, so I removed that logic. But if that's desirable, we can keep it. I added back that stuff and just corrected the paths in b846dbd93ee2c0efb8df2914782991653ae7c644.